src/Expander.hs

```diff
@@ -483,7 +485,7 @@ initializeKernel = do
             exprDest <- liftIO $ newSplitCorePtr
             bind b (EIncompleteDefn var x exprDest)
             schPtr <- liftIO $ newSchemePtr
-            linkDecl dest (Define x var schPtr exprDest)
+            linkOneDecl dest (Define x var schPtr exprDest)
```

Comment:
`linkDecl` still exists and still links a `DeclPtr` to a `Decl`. `linkOneDecl` is a version which binds a `ModBodyPtr` to a `Decl`. It is defined in terms of `linkDecl` and `linkModBody`.

Reply:
Sounds reasonable to me!